src/Tizen.NUI/src/internal/Utility/ColorCutQuantizer.cs

```diff
@@ -0,0 +1,633 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Modified by Woochan Lee(wc0917.lee@samsung.com)
+ */
+
+using System;
+using System.Drawing;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Tizen.NUI
+{
+    internal sealed class ColorCutQuantizer
+    {
+        private const float BlackMaxLightness = 0.05f;
```

Comment:
private variables have to start with a lowercase letter.
'BlackMaxLightness' -> 'blackMaxLightness'

please check other private variables as well